tests/api/test_auth.py

```diff
@@ -48,7 +59,12 @@ def test_api_auth_model_serveruserscredentials():
         users += ServerUsersCredentials.parse_obj(
             [
                 UserCredentials(
-                    username="foo", hash="notsorealhash", scopes=["foo_scope"]
+                    username="foo",
+                    hash="notsorealhash",
+                    scopes=["foo_scope"],
+                    agent={"mbox": "mailto:foo2@example.com"},
                 ),
             ]
         )
+
+    # TODO: should we verify agent uniqueness ?
```

Comment:
I think we should!

Reply:
It was decided not to do this in the end.